{"uuid": "07a2eb7a-fd90-4c9b-8722-2702ed5b7166", "children": ["3d1212c0-b626-4e1f-9749-bdb5d2875acf"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'launch_app'\n", "trace": "  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\pc\\Desktop\\Challenge Gaston Ferreyra\\Mobile-test\\conftest.py\", line 41, in driver\n    driver_instance.launch_app()  # inicia la app\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1759531254103, "stop": 1759531258622}], "start": 1759531254103, "stop": 1759531258695}